### DISCUSSION TITLE
Can I download copies of fics from multiple lists?

### DISCUSSION THREAD

**Discussion by IceRed-Drone** [nianeyna/ao3downloader#106] [Q&A] [answered]
For example, I want to make an archive of all the fics in a tag, but also have a separate folder of the ones in my bookmarks. Is the project set up to recognize this? Alternatively, if I'm understanding how this works correctly, could I copy the log file for the tag, delete the old one, run my bookmarks through the downloader, then replace the new log file with the one for the tag again?

**Comment by nianeyna** [comment, marked as answer]
Yep, you can definitely shuffle the log files around like that. The downloader can't sort things into folders automatically, but you can download a tag, move the results into a separate folder (along with their log file, if you like) then start over with your bookmarks, and move that into a separate folder, etc.
